docs/bundler.md

```diff
@@ -71,6 +71,9 @@ works correctly:
 - BUNDLE_DEPLOYMENT: "true"
 - BUNDLE_NO_PRUNE: "true"
 - BUNDLE_VERSION: "system"
+- BUNDLE_ALLOW_OFFLINE_INSTALL: "true"
```

Comment:
I just noticed this needs to be updated in the design as well: https://github.com/containerbuildsystem/cachi2/blob/main/docs/design/bundler.md#other-configuration-that-was-considered

> BUNDLE_ALLOW_OFFLINE_INSTALL is not working either with bundle install for some reason, which could be probably the most logical solution in this case.

I don't know what the version of Bundler was back then, but this variable doesn't seem to be causing any issues with default Bundler on either Ruby 3.3 or 3.2, so I'd just drop that hunk and add the variable to the list.